### ISSUE TITLE
Pyjs dispatcher

### ISSUE THREAD

**Issue by maho** [pyjs/original-pyjs#747]
Regarding discussion at https://groups.google.com/d/msg/pyjs-users/-JHbPk5RFH8/IWLk-KrQqPsJ - I have implemented pyjs dispatcher - which allows to reach py code from deep js. 

I'm not sure what's the proper way of making change to code under new government in project, so I made (temporary?) fork of project at bitbucket and pushed change there. 

Project is at https://bitbucket.org/maho/pyjs-changes.git and change: https://bitbucket.org/maho/pyjs-changes/changeset/4de7bd72a108

**Comment by bittner** [comment]
You should fork the pyjs repo here on GitHub, apply the changes, and make a Pull Request to the pyjs repository. You can most probably create a patch from your changes on BitBucket, and apply that to a new fork here on GitHub. Good luck!

**Comment by maho** [comment]
W dniu 28.06.2012 07:20, bittner pisze:

> You should fork the pyjs repo here on GitHub, apply the changes,
> and make a Pull Request to the pyjs repository.

Ah, ok.  I did it in such way.

**Comment by anthonyrisinger** [comment]
see #748

tldr; this can go in wiki only.
